src/Microsoft.AspNet.IISPlatformHandler/IISPlatformHandlerMiddleware.cs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.AspNet.IISPlatformHandler
 {
     public class IISPlatformHandlerMiddleware
     {
-        private const string XIISWindowsAuthToken = "X-IIS-WindowsAuthToken";
+        private const string MSPlatformHandlerWinAuthTOken = "MS-PLATFORM-HANDLER-WINAUTHTOKEN";
```

Comment:
Need back compat or we'll break everyone at the same time. Read both headers for now.